docs/workflows/quick-start-guide.md

```diff
@@ -1,52 +1,131 @@
+| --profile    | G3T_PROFILE     | gen3-client profile | production    |
+| --format     | G3T_FORMAT      | Output format.      | default: yaml |
+
+Alternatively, you can set the environmental variables using the `EXPORT` function e.g.:
+
+```sh
+export G3T_PROJECT_ID=aced-myproject
+```
+
+
 ## 1. Upload Data (To an Existing Project on ACED IDP)
-This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarily to `git` commands
+This section outlines how to add data to ACED IDP. Many `g3t` commands behave similarly to `git` commands
+
+### Initialize a new project:
 
-Initialize a new project
 ```
 g3t init aced-demo
 ```
 
-Clone an existing project
-```
-g3t clone aced-demo
-```
+* Similar to `git init`, this command initializes a new project in the current directory
+* The id is `aced-demo`; the program name is `aced`, the project name is `demo`
+* `Program name` is predefined by the institution, the `g3t ping` command will show the available programs
+* `Project name` must be unique within the server, and must be alphanumeric and contain no spaces
+
+
+### Add file(s):
 
-Add file(s)
 ```
 g3t add folder/file.tsv
 g3t add folder/file2.tsv
 ```
 
-Create meta data for your files. This `g3t` command will take care of creating this for you
+* This command will record an entry for each file in the `MANIFEST` directory.
+* Optionally you can tag each file with patient, specimen or other identifiers. See `g3t add --help` for more information.
+
+
+### Create meta data for your files. The `g3t meta init` command will take care of creating FHIR compliant data for you.
+
 ```
-g3t utilities meta create
+g3t meta init
 ```
 
-Check that the expected files are queued for upload
+* This command will create metadata files in the META directory. At a minimum, the directory will contain:
+
+| file                     | contents                   |
+|--------------------------|----------------------------|
+| ResearchStudy.ndjson     | Description of the project |
+| DocumentReference.ndjson | File information           |
+
+Additional metadata files for patient, specimen, and other entities will be generated based on options provided to the `add` command.
+
+| file                   | contents               |
+|------------------------|------------------------|
+| Patient.ndjson         | Patient information    |
+| ResearchSubject.ndjson | Enrollment information |
+| Specimen.ndjson        | Sample information     |
+
+
+> Note: You may supply your own FHIR data, the `g3t meta init` command is a convenience method to generate the required files.
+
+### Check that the metadata is valid:
+
+```
+g3t meta validate
+```
+
+
+### Check that the expected files are queued for upload
 ```
 g3t status
 ```
 
-Commit files
+### Commit files
 ```
 g3t commit -m "Adding files"
 ```
 
-Push to ACED IDP
+### Push to ACED IDP
 ```
 g3t push
 ```
 
-View the upload status (pending or complete)
+Notes:
```

Comment:
Bullet points below not getting formatted, might just need to add a new line here